db/migrate/20141204185232_add_force_password_reset.rb

```diff
@@ -0,0 +1,9 @@
+class AddForcePasswordReset < ActiveRecord::Migration
+  def up
+    add_column :users, :force_password_reset, :boolean, :default => true
```

Comment:
It could be better to have default set to true and set already existing users to false so they are not prompted for new password. It should be also considered to move the `default: true` setting to User initialize method instead so the domain logic of the user is together in the model definition without parts defined in DB schema.